packages/flutter_tools/lib/src/test/flutter_web_goldens.dart

```diff
@@ -98,14 +98,14 @@ class TestGoldenComparator {
     return _processManager!.start(command, environment: environment);
   }
 
-  Future<String?> compareGoldens(Uri testUri, Uint8List bytes, Uri goldenKey, bool? updateGoldens) async {
+  Future<String?> compareGoldens(Uri testUri, Uint8List bytes, Uri goldenKey, bool? updateGoldens, Map<String, dynamic>? customProperties) async {
```

Comment:
What is `customProperties` for?

Reply:
For any custom properties needed by the custom comparator supplied by `flutter_test_config.dart`. In the case of Flutter's own goldens the only custom property is `isFlaky`. Without it, I'd have to hard-code the notion of `isFlaky` into the tool. Since we decided to limit this concept to Flutter's own goldens, I had to add a more generic way of passing custom properties. I'm open to other suggestions.